src/ContentHubClient.php

```diff
@@ -305,17 +309,16 @@ public function getEntities(array $uuids)
                 ],
               ],
             ];
-            $options['body'] = json_encode((array)$query);
-            $results = $this->getResponseJson($this->get('_search', $options));
-            if (is_array($results) && isset($results['hits']['total']) && $results['hits']['total'] > 0) {
+            $options['body'] = json_encode($query);
+            $results = self::getResponseJson($this->get('_search', $options));
+            if (isset($results['hits']['total'])) {
```

Comment:
In general I prefer !empty() to isset(). We can do that in a follow up though.

Reply:
There are looooooooooots of other places that need such changes. Now, with having so many tests in place, it'll be like a breeze to refactor and make it better.